src/db/rss-parser.ts

```diff
@@ -1,51 +1,50 @@
 import Parser from 'rss-parser';
 import cheerio from 'cheerio';
 import { Article } from '../entities/Article';
-import { PublicationModel } from '../entities/Publication';
+import { Publication } from '../entities/Publication';
 import PublicationRepo from '../repos/PublicationRepo';
 
 const getRecentArticles = async (): Promise<Article[]> => {
   const parser = new Parser();
 
-  const nameToSlugMap = {};
```

Comment:
Unfortunately, this script is now no longer being used. Your work on this PR has been absolutely incredible though!!

Reply:
And will be implemented in some form!